cloudaux/orchestration/aws/s3.py

```diff
@@ -17,14 +17,57 @@
 from cloudaux.orchestration import modify
 
 from botocore.exceptions import ClientError
+from bunch import Bunch
 import logging
 import json
 
 
 logger = logging.getLogger('cloudaux')
 
 
-def get_grants(bucket_name, include_owner=False, **conn):
+FLAGS=Bunch(
+    GRANTS=1,
+    GRANT_REFERENCES=1,
+    OWNER=1,
+    LIFECYCLE=2,
+    LOGGING=4,
+    POLICY=8,
+    TAGS=16,
+    VERSIONING=32,
+    WEBSITE=64,
+    CORS=128,
+    NOTIFICATIONS=256,
+    ACCELERATION=512,
+    REPLICATION=1024,
+    ANALYTICS=2048,
+    METRICS=4096,
+    INVENTORY=8192,
+    CREATED_DATE=16384,
+    ALL=32767)
+
+
+class FlagRegistry:
+    r = list()
+
+    @classmethod
+    def register(cls, flag, key):
+        def decorator(fn):
+            flag_list = flag
+            key_list = key
+            if type(flag) not in [list, tuple]:
+                flag_list = [flag] 
+            if type(key) not in [list, tuple]: 
+                key_list = [key]
+            for idx in xrange(len(flag_list)):
+                cls.r.append(dict(flag=flag_list[idx], key=key_list[idx], method=fn, rtv_ix=idx))
```

Comment:
Maybe add a comment as simple as an example line here just as a shortcut to future reviewers?